Mathlib/Geometry/Manifold/ContMDiff/Atlas.lean

```diff
@@ -106,9 +109,9 @@ theorem contMDiffWithinAt_extChartAt_symm_range
   (contMDiffWithinAt_extChartAt_symm_target x hy).mono_of_mem_nhdsWithin
     (extChartAt_target_mem_nhdsWithin_of_mem hy)
 
-/-- An element of `contDiffGroupoid ⊤ I` is `C^n` for any `n`. -/
+/-- An element of `contDiffGroupoid n I` is `C^n` for any `n`. -/
```

Comment:
Please update this doc-string: "for any n" is now false. (I haven't checked if other doc-strings in this file are also stale now.)